project/Dependencies.scala

```diff
@@ -46,11 +45,12 @@ object Dependencies {
   val scalaCheck = "org.scalacheck" %% "scalacheck" % "1.11.5" % "test"
   val scalajTime = "org.scalaj" % "scalaj-time_2.10.2" % "0.7"
   val scalaTest = "org.scalatest" %% "scalatest" % "2.2.1" % Test
+  val scalaz = "org.scalaz" %% "scalaz-core" % "7.0.6"
```

Comment:
it wasn't me! whoever started using scalaz without adding a direct dependency, tut tut.